src/shared-components/Navbar/Navbar.spec.tsx

```
@@ -72,7 +74,12 @@ vi.mock('shared-components/DropDownButton/DropDownButton', () => ({
         <button type="button" data-testid={`${dataTestIdPrefix}-toggle`}>
           {label}
         </button>
-        <div data-testid={`${dataTestIdPrefix}-icon`}>{icon}</div>
+        <div
+          data-testid={`${dataTestIdPrefix}-icon`}
+          data-icon-type={type}
+        >
```

ESLint

[error] Replace `⏎··········data-testid={`${dataTestIdPrefix}-icon`}⏎··········data-icon-type={type}⏎········` with `·data-testid={`${dataTestIdPrefix}-icon`}·data-icon-type={type}`

(prettier/prettier)